### ISSUE TITLE
NTR: GAMT enzyme activity low in fibroblasts or lymphocytes (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8208]
New term request GAMT enzyme activity low in fibroblasts or lymphocytes (segal_090513171231): Guanidinoacetate methyltransferase  UMLS: 2748618

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000722
- label: Reduced tissue guanidinoacetate methyltransferase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:15651030
- def: Activity of the mitochondrial enzyme guanidinoacetate methyltransferase in tissues below the lower limit of normal. The activity can be measured in multiple tissues including culutured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8208
- synonyms:
